### PR TITLE
Multi-Con Vote App

### PR DESCRIPTION
**Pull request by zfox23**
Collab with @robin-k-wilson!

Released script S3 link:
https://hifi-content.s3.amazonaws.com/Experiences/Releases/events/multicon/multiConApp/2019-03-13_15-11-00/multiConApp/multiConApp_app.js

Uses production database (we'll clear out any votes accrued during testing before the event).

Go to https://highfidelity.co/multiConVote/?type=getLeaderboard to see the current leaderboard.

Votes cast after 2pm on the day of MultiCon (3/16) won't count in the leaderboard.

### CODE REVIEW

File: events/multiCon/multiConApp/multiConApp_app.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                });
+            }
+        });
+    }
+
+
+    function vote(usernameToVoteFor) {
+        var myUsername = AccountServices.username;
+        if (myUsername === "Unknown user") {
+            console.log("User tried to vote, but is not logged in!");
```
Comment (RebeccaStankus) [inline events/multiCon/multiConApp/multiConApp_app.js]: Did you meant to leave this?

File: events/multiCon/webApps/multiConVote/multiConVote.js [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+//
+
+var http = require('http');
+var url = require('url');
+var dbInfo = require('./dbInfo.json');
+
+
+// Gets all information associated with users participating in the Best Avatar Contest
+function getParticipants(voterUsername, response) {
+    // have they voted?
```
Comment (RebeccaStankus) [inline events/multiCon/webApps/multiConVote/multiConVote.js]: maybe remove these too?

**Comment by AndySeattle** [comment]
Oh this was not marked as passed, must of been an oversight, Salina tested this then i did also.

File: events/multiCon/multiConApp/multiConApp_app.js (hunk 1 above)
Comment (robin-k-wilson) [reply events/multiCon/multiConApp/multiConApp_app.js]: removed

**Pull request merged**